src/WebJobs.Script.Grpc/WebJobs.Script.Grpc.csproj

```diff
@@ -25,13 +25,13 @@
     <PackageReference Include="Grpc.Tools" Version="2.27.0">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0-preview.3.21201.4" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.0-preview.3.21201.4" />
```

Comment:
@fabiocav -- do you recall why we never bumped these to 3.1? Was it b/c of interaction with customer in-proc code? Wondering if I should keep all of this back on 2.2.

Reply:
Yes, it was because of the migration requirements from 2.0 - 3.0. 

We can keep your changes for now